chid.pxd

```diff
@@ -1,6 +1,6 @@
 from libc.stddef cimport wchar_t, size_t
 
-cdef extern from "hidapi.h":
+cdef extern from "hidapi/hidapi.h":
```

Comment:
conceptually - this isn't right
one should always include `<hidapi.h>` in the source code
and at the build/configuration level a correct include path should be added using one of the package search tools